src/Cohesity.Powershell/Scripts/ProtectionJobRun/Update-CohesityProtectionJobRun.ps1

```diff
@@ -0,0 +1,111 @@
+#### USAGE ####
+#	Update-CohesityProtectionJobRun -ProtectionJobName viewJob -JobRunId 65675,65163 -ExtendRetention 10
+#	Update-CohesityProtectionJobRun -ProtectionJobName viewJob -JobRunId 65675 -ExtendRetention 10
+#	Update-CohesityProtectionJobRun -ProtectionJobName viewJob -ExtendRetention 10
```

Comment:
Also add starttimeusecs and endtimeusecs as parameter..just like in UI , we can filter job runs between a date range.

Reply:
Fixed, filter based on the start and end time to update the retention